samples/FakeItEasy.Examples.ruleset

```diff
@@ -15,4 +14,4 @@
     <Rule Id="CA2201" Action="None" />
     <Rule Id="CA2210" Action="None" />
   </Rules>
-</RuleSet>
\ No newline at end of file
+</RuleSet>

```

Comment:
Whoops. I misread your comment (and the diff). I thought you were upset that the CRLF went away, not that they were added. Sorry.
Yes, adding an exception seems like a good idea.
Although it might not have made a difference in this case, as I can't promise I used VS as an editor. May have been emacs. I honestly do not remember.